esinstall/src/rollup-plugins/rollup-plugin-strip-source-mapping.ts

```diff
@@ -9,7 +9,9 @@ export function rollupPluginStripSourceMapping(): Plugin {
   return {
     name: 'snowpack:rollup-plugin-strip-source-mapping',
     transform: (code) => ({
-      code: code.replace(/[^'"`]\/\/+#\s*sourceMappingURL=.+$/gm, ''),
+      code: code
+        // [a-zA-Z0-9-_\*?\.\/\&=+%]: valid URL characters (for sourcemaps)
+        .replace(/\/\/#\s*sourceMappingURL=[a-zA-Z0-9-_\*\?\.\/\&=+%\s]+$/gm, ''),
```

Comment:
ah gotcha, so now youre testing that nothing non-URL comes afterwards in the line? Smart!

only random thing that I can think of: can this line end with a semicolon?

Reply:
I don‘t think so? Because it‘s a line comment. And a sourcemap URL probably won‘t have a semicolon in it right?

Reply:
yea, I don't think so either